packages/resourcehacker.vm/resourcehacker.vm.nuspec

```diff
@@ -2,12 +2,12 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>resourcehacker.vm</id>
-    <version>5.1.8</version>
+    <version>5.2.4</version>
     <authors>Angus Johnson</authors>
     <description>Resource Hacker is a resource editor for 32bit and 64bit Windows applications.</description>
     <dependencies>
       <dependency id="common.vm" />
-      <dependency id="resourcehacker.portable" version="[5.1.8]" />
+      <dependency id="reshack" version="[5.2.4]" />
```

Comment:
prefer this or https://community.chocolatey.org/packages/reshack.portable?

Reply:
Oh I missed this, good catch! I'll update b/c I'd prefer the portable version